airbyte-integrations/connectors/source-shopify/unit_tests/unit_test.py

```diff
@@ -5,13 +5,14 @@
 
 import pytest
 import requests
-from source_shopify.source import BalanceTransactions, DiscountCodes, FulfillmentOrders, PriceRules, ShopifyStream, SourceShopify
+from source_shopify.source import BalanceTransactions, DiscountCodes, FulfillmentOrders, PriceRules, SourceShopify
```

Comment:
To verify deletion logic, can you add unit testing which mocks API behavior?

Reply:
For sure!